testing/bcc/APKBUILD

```diff
@@ -1,17 +1,28 @@
 # Maintainer: Adam Jensen <acjensen@gmail.com>
 pkgname=bcc
-pkgver=0.8.0
+pkgver=0.9.0
 pkgrel=0
 pkgdesc="A toolkit for creating efficient kernel tracing and manipulation programs"
 url="https://github.com/iovisor/bcc/"
-arch="aarch64 x86_64"
+arch="aarch64 x86 x86_64"
 license="Apache-2.0"
 # bcc's test suite requires privileged access to run BPF programs
 options="!check"
```

Comment:
Did you try `options="checkroot"` ? -https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#options

Reply:
The test suite runs BPF programs in the kernel, so I don't think fakeroot will help here.